### PR TITLE
Fix 2 issues

### LINKED ISSUE #17: Say command doesnt copy mentions and emotes values

### LINKED ISSUE #26: Disconnecting from afk channel is dependent on log channel
If log channel isn't set, bot won't disconnect user on joining afk channel.

### PR DESCRIPTION
**Pull request by szyjah**
Kick ppl from Under The Bridge and make bot say whole sentences.
Resolves #26 
Resolves #17

### CODE REVIEW

**Comment by szyjah** [comment]
Resolves #26

**Review by seker212 — CHANGES_REQUESTED** [review]
This does not resolves #17

**Comment by seker212** [comment]
@szyjah Also please use [Guild.afk_channel](https://discordpy.readthedocs.io/en/stable/api.html?highlight=guild#discord.Guild.afk_channel) to get current afk channer. Just use a check if `after` channel is guild's afk channel

**Comment by szyjah** [comment]
Apllied suggested change.
Regarding #17 : I revorted wrong commit. Upsie.

File: stuff.py [hunk 1]
```diff
@@ -152,10 +152,10 @@ async def on_voice_state_update(self ,member, before, after):
                     else:
                         await self.channel.send(ts+'***'+user+'*** hides himself in video call')
                 
-                #No afk channel policy
-                if after.channel != None:
-                    if after.channel.name == 'Pod mostem':
-                        await member.move_to(None,reason="OOF")
+        #No afk channel policy
+        if after.channel != None:
+            if after.channel == after.channel.guild.afk:
```
Comment (Niewidzialny84) [inline stuff.py:157]: Tested? Beacause i can doubt this can not work and i did not see a method there that works like this.

**Review by Niewidzialny84 — CHANGES_REQUESTED** [review]

File: stuff.py (hunk 1 above)
Comment (szyjah) [reply stuff.py]: upsie

**Review by Niewidzialny84 — APPROVED** [review]
Looks nice now

**Review by seker212 — APPROVED** [review]

**Pull request merged**